### PR TITLE
Initialized the db contexts

### PR DESCRIPTION
**Pull request by TermanEmil**
Initialized ApplicationDbContext with Audio and AudioMeta.

Initialized the AuthenticationDbContext for Identity.

I managed to manually test AuthenticationDbContext

### CODE REVIEW

File: NPlaylist/db_migrate_and_update.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# This script creates migrations and updates the database
+# Execute these commands in the project containing the target db context
+
+# ../NPlaylist/ - project where the db context is added (with Startup.cs)
+# <target db context> - the db context: e.g. ApplicationDbContext
+
+dotnet ef migrations add -v <migration name> -s ../NPlaylist/ --context <target db context>
```
Comment (aivascu) [inline NPlaylist/db_migrate_and_update.sh]: not sure the other guys will be able to run Shell scripts on their Windows machines (maybe inside git shell...), but OK

Comment (TermanEmil) [reply NPlaylist/db_migrate_and_update.sh]: I'll change that

Comment (georgewwww) [reply NPlaylist/db_migrate_and_update.sh]: I checked and can run these commands inside Package Manager Console

File: NPlaylist/NPlaylist.Persistance/NPlaylist.Persistance.csproj [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```
Comment (aivascu) [inline NPlaylist/NPlaylist.Persistance/NPlaylist.Persistance.csproj]: Change the target framework to the minimum necessary version of netstandard

File: NPlaylist/NPlaylist.Authentication/NPlaylist.Authentication.csproj [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```
Comment (aivascu) [inline NPlaylist/NPlaylist.Authentication/NPlaylist.Authentication.csproj]: Change the target framework to the minimum necessary version of netstandard

**Pull request merged**